### PR TITLE
Update CHANGELOG and bump Release.toml for v1.0.5

### PR DESCRIPTION
**Pull request by etungsten**
**Description of changes:**
Prepares for v1.0.5 release.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
...
+Attempts to downgrade may result in inconsistencies between ECS and the Bottlerocket ECS instance.
+
+## OS Changes
+
+* Add high-level update subcommands to apiclient ([#1219], [#1232])
+* Add kernel lockdown settings ([#1223])
+* Add restart-commands for docker, kubelet, containerd ([#1231], [#1262], [#1258])
+* Add proper restarts for host-containers ([#1235], [#1230], [#1258])
+* Fix SELinux policy ([#1236])
+* Stop creating container tasks under host-container's cgroup ([#1242])
```
Comment (tjkirch) [inline CHANGELOG.md]: I think this one could go in the list for "Add proper restarts for host-containers" to indicate that it was intimately tied to that feature.  If you think it's important to call out the specific change here without having to click through (if it has impacts for any other reason) then perhaps we could make it a sub-bullet?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,64 @@
...
+* Set version and revision strings for containerd ([#1248])
+* Add host-container user-data setting ([#1244], [#1247])
+* Add network proxy settings ([#1204], [#1262], [#1258])
+* Update kernel to 5.4.80-40.140 ([#1257])
+* Add aws-k8s-1.19 variant with Kubernetes 1.19 ([#1256])
```
Comment (tjkirch) [inline CHANGELOG.md]: I'd probably put this one at the top.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,64 @@
...
+* Update third-party software packages ([#1264])
+* Update Rust dependencies ([#1267])
+* Update ecs-agent to 1.48.1 ([#1201])
+
+## Build Changes
+
+* Improve support for out-of-tree kernel modules ([#1220])
+* Fix message in partition side check condition ([#1233], **thanks @pranavek!**)
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Fix message in partition size check condition ([#1233], **thanks @pranavek!**)
```

**Review by tjkirch — APPROVED** [review]

**Comment by etungsten** [comment]
Push above addresses @tjkirch 's comments.

**Review by tjkirch — APPROVED** [review]

**Review by webern — APPROVED** [review]

**Comment by etungsten** [comment]
Push above rebases on top of the last few changes for v1.0.5

**Review by zmrow — APPROVED** [review]
Good work!

:partying_face:

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,63 @@
+# v1.0.5 (2021-01-15)
+
+**Note for aws-ecs-1 variant**: due to a change in the ecs-agent's data store schema, the aws-ecs-1 variant cannot be downgraded after updating to v1.0.5.
```
Comment (samuelkarp) [inline CHANGELOG.md]: nit: `ecs-agent` is the name of the package but "ECS agent" is the name of the software

```suggestion
**Note for aws-ecs-1 variant**: due to a change in the ECS agent's data store schema, the aws-ecs-1 variant cannot be downgraded after updating to v1.0.5.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,63 @@
...
+Attempts to downgrade may result in inconsistencies between ECS and the Bottlerocket ECS instance.
```
Comment (samuelkarp) [inline CHANGELOG.md]: nit: "container instance" is the name used for a member of an ECS cluster

```suggestion
Attempts to downgrade may result in inconsistencies between ECS and the Bottlerocket container instance.
```

**Comment by etungsten** [comment]
Push above addresses @samuelkarp 's comments.

**Review by samuelkarp — APPROVED** [review]

**Pull request merged**